cmd/analyze/analyze.go

```diff
@@ -36,10 +36,11 @@ var AnalyzeCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
-		err = config.RunAnalysis()
-		if err != nil {
-			color.Red("Error: %v", err)
-			os.Exit(1)
+		analysisErrors := config.RunAnalysis()
+		if len(analysisErrors) != 0 {
+			for _, err := range analysisErrors {
+				color.White("Error: %s", err)
```

Comment:
We might want another colour as that won't show on default terminals

Reply:
The Kubernetes client returns this kind of warning if we try to parse a deprecated APIVersion. I chose this color to maintain the same style, but I can change it
![Screenshot 2023-04-20 at 14 10 27](https://user-images.githubusercontent.com/99146727/233361969-98ffec10-2962-4cf4-9e8c-f845f56b1855.png)